elizabeth/homework1.js

```diff
@@ -0,0 +1,173 @@
+/* Section 1
+a) True -- == is not exact and JS will auto convert "1" to a number.
+
+b) False -- === is exact, the values must match in type and value.
+
+c) True -- for the same reason "a" was true, both value and type are not checked, however it does not actually return a value.
+
```

Comment:
In this instance, x == 'x' returns false. The reason for this is that variable x does not necessarily return the same value a string 'x'. For example, variable x could contain 'giraffe' which is not equivalent to 'x'.